internal/ls/completions.go

```diff
@@ -4193,7 +4202,7 @@ func (l *LanguageService) createLSPCompletionItem(
 		InsertTextFormat: insertTextFormat,
 		TextEdit:         textEdit,
 		CommitCharacters: commitCharacters,
-		Data:             nil, // !!! auto-imports
+		Data:             &data,
```

Comment:
This will eventually come to bite us, but one thing we are not doing at all in the LS at the moment is respecting the _client's_ capabilities. All of these things should actually be things where we check the client's caps and say "do they support resolve?", and if not,  include all of this information like docs up front, or similar.

Not really critical for this PR, but something to think about.

Reply:
Yep, that's what I meant to note in the description:
>Note that for now, if a client doesn't support lazy computation of detail and documentation, we simply won't fill in those properties.
Especially for auto-imports, this could be detrimental to performance.